rice-middleware/ksb/client-impl/src/main/java/org/kuali/rice/ksb/security/soap/KualiPasswordEncryptor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2006-2015 The Kuali Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.opensource.org/licenses/ecl2.php
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kuali.rice.ksb.security.soap;
+
+import org.apache.wss4j.common.crypto.PasswordEncryptor;
+
+public class KualiPasswordEncryptor implements PasswordEncryptor {
+    @Override
+    public String encrypt(String password) {
+        return password;
+    }
+
+    @Override
+    public String decrypt(String encryptedPassword) {
+        return encryptedPassword;
+    }
+}

```

Comment:
What has changed that we need a PasswordEncryptor?  Is the password being stored or submitted somewhere?

Also what is the implications of us supplying an implementation that doesn't encrypt passwords?  (i.e. will institutions end up using this implementation and not be aware that passwords are never encrypted?)